spec/request/user_request_spec.rb

```diff
@@ -6,6 +6,28 @@
 RSpec.describe 'Fetching user data', type: :request do
   include SchemaMatchers
 
+  context 'GET /v0/user in camel - when an LOA 3 user is logged in' do
+    let(:mhv_user) { build(:user, :mhv) }
+
+    before do
+      allow_any_instance_of(MhvAccountTypeService).to receive(:mhv_account_type).and_return('Premium')
+      mhv_account = double('MhvAccount', creatable?: false, upgradable?: false, account_state: 'upgraded')
+      allow(MhvAccount).to receive(:find_or_initialize_by).and_return(mhv_account)
+      allow(mhv_account).to receive(:user_uuid).and_return(mhv_user.uuid)
+      allow(mhv_account).to receive(:terms_and_conditions_accepted?).and_return(true)
+      allow(mhv_account).to receive(:needs_terms_acceptance?).and_return(false)
+      allow(mhv_account).to receive(:user=).and_return(mhv_user)
+      create(:account, idme_uuid: mhv_user.uuid)
+      sign_in_as(mhv_user)
+      get v0_user_url, params: nil, headers: { 'X-Key-Inflection' => 'camel' }
+    end
```

Comment:
Could we refactor this and the nearly identical block below into a common context + before block, using a variable for the key-inflection header?

Reply:
I don't like giant `before`-blocks and I especially don't like using `lets` to pass in values to a `before`-block.  So I thought duplicating the `before`-block was preferable to that approach.  I'll take another look and see if I can refactor it in a way that's clearer

Reply:
I hated it but, I went ahead and used the existing context